aiven/client/cli.py

```diff
@@ -1210,6 +1210,94 @@ def service_es_acl_del(self):
             username=self.args.username, del_rules=self.args.rule)
         print(response.get("message"))
 
+    @arg.project
+    @arg.service_name
+    def service_connector_available(self):
```

Comment:
One more thing: I think all of these methods should be named `service__connector__<action>` to group them under `service connector` instead of polluting the `service` command with a ton of sub commands.